content/src/draw-protractor.js

```diff
@@ -3,11 +3,11 @@
 // See: http://jsfiddle.net/jamessynge/QQ43x/
 
 var $   = require('jquery'),
+  jqueryturtle  = require('jquery-turtle'),
     see = require('see');
 
-
+var t;
```

Comment:
Is this the right place for this scoping? Looks like it ought to be inside the function where it's used.